Makefile.template

```diff
@@ -142,20 +150,24 @@ distclean:
 $(WORKSPACE):
 	mkdir -p $@
 
+prepare_buildtool_internal: $(WORKSPACE)
+	cmake -S infra/buildtool -B $(BUILDTOOL_WORKSPACE)/obj -DBUILDTOOL_PATH=$(BUILDTOOL_PATH)
+	cmake --build $(BUILDTOOL_WORKSPACE)/obj/ -j$(NPROCS)
 prepare_nncc_internal: $(WORKSPACE)
-ifneq ($(CROSS_BUILD),1)
-	./nncc configure -DBUILD_GTEST=OFF -DENABLE_TEST=OFF -DEXTERNALS_BUILD_THREADS=$(NPROCS) -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) \
+ifeq (,$(findstring android,$(TARGET_OS)))
```

Comment:
If `TARGET_OS` contains `android` string, will the following function be executed?

Reply:
No. If `TARGET_OS` does not contain.

Reply:
Ah, I got it. This expression is compare '' string with findstring. Thank you :)